src/panel/poi/PoiPanel.jsx

```diff
@@ -268,28 +252,22 @@ export default class PoiPanel extends React.Component {
         text: _('Back to list'),
         className: 'poi_panel__back_to_list',
       };
-    } else {
-      backAction = {
-        callback: this.backToSmall,
-        text: _('Back'),
-        className: 'poi_panel__back_mobile',
-      };
     }
 
-    const header = <div className="poi_panel__header">
-      {backAction &&
-        <div className={backAction.className} onClick={backAction.callback}>
-          <i className="poi_panel__back icon-arrow-left" />
-          <span className="poi_panel__back_text">{backAction.text}</span>
-        </div>
-      }
-      {isFromPagesJaunes(poi) && <img className="poi_panel__pj_logo"
-        src="./statics/images/pagesjaunes.svg"
-        alt="PagesJaunes" />
-      }
-    </div>;
+    const header = backAction &&
+      <div className={backAction.className} onClick={backAction.callback}>
```

Comment:
Maybe it's possible to use the `<Flex>` component here to get rid of even more CSS.

Reply:
yep, it's done, thanks